external-import/intel471_v2/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.12-alpine
+ENV CONNECTOR_TYPE=EXTERNAL_IMPORT
+
+COPY src /opt/opencti-intel471
+
+# hadolint ignore=DL3003
+RUN apk update && apk upgrade && \
+    apk --no-cache add git build-base libmagic libxslt libxslt-dev libxml2 libxml2-dev && \
+    cd /opt/opencti-intel471 && \
```

Comment:
Could you change it and add v2 please to not having problem with v1 of connector ?